### ISSUE TITLE
Serialized TodoTask is not understood by server

### ISSUE THREAD

**Issue by gaspik** [microsoftgraph/msgraph-beta-sdk-dotnet#495]
**Describe the bug**
TodoTask is serialized in a way that the server does not understand it

**To Reproduce**
Create new TodoTask and try to add it to existing task list. Example code:

```
var todoTask = new TodoTask
			{
				Title = "SampleTitle",
				Body = new ItemBody()
				{
					ContentType = BodyType.Html,
					Content = "SampleHtml"
				}
			};

await GraphServiceClient.Me
					.Todo
					.Lists[uniqueId]
					.Tasks
					.PostAsync(todoTask , cancellationToken: cancellationToken);
```

This gets serialized as following:

```
{
    "@odata.type": "#microsoft.graph.todoTask",
    "body": {
        "content": "SampleHtml",
        "contentType": "html",
        "@odata.type": "#microsoft.graph.itemBody"
    },
    "title": "SampleTitle"
}
```

And server replies with
```
{
    "error": {
        "code": "invalidRequest",
        "message": "One or more errors occurred. (One or more errors occurred. (A type named 'microsoft.toDo.todoTask' could not be resolved by the model. When a model is available, each type name must resolve to a valid type.))",
        "innerError": {
            "code": "InvalidModel",
            "date": "2022-07-20T13:52:13",
            "request-id": "e4ad2f1c-a189-4ee5-80ef-0124626352d1",
            "client-request-id": "e4ad2f1c-a189-4ee5-80ef-0124626352d1"
        }
    }
}
```

Removing both odata.type resolves the issue. This was working in 5.8.0-preview, so I assume this has to do with https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/pull/493

**Expected behavior**
TodoTask is properly created

**Client version**
5.10.0-preview

**Desktop (please complete the following information):**
- OS: Win 10 x64
- Browser: N/A, WinForms application using standard HttpClient
- Version: N/A

**Comment by martin-ixzal-com** [comment]
Also the same issue in 5.13.0-preview.

          var newTodoTask = new TodoTask
          {
            Title = NEW_TODOTASK_TITLE,
            Body = new ItemBody { Content = "-", ContentType = BodyType.Text },
          };

          newTodoTask = await graphClient.Me.Todo.Lists[taskList.Id].Tasks.PostAsync(newTodoTask).ConfigureAwait(true);   


ex.Message: Exception of type 'Microsoft.Graph.Beta.Models.ODataErrors.ODataError' was thrown.

ex.Source: Microsoft.Kiota.Http.HttpClientLibrary
ex.Error.Message: One or more errors occurred. (One or more errors occurred. (A type named 'microsoft.toDo.todoTask' could not be resolved by the model. When a model is available, each type name must resolve to a valid type.))

**Comment by andrueastman** [comment]
Thanks for your patience with this issue. 

This should now be resolved with the latest [5.14.0-preview](https://www.nuget.org/packages/Microsoft.Graph.Beta/5.14.0-preview) version
